app/controllers/users_controller.rb

```diff
@@ -17,6 +17,13 @@ def create
 
     if @user.save
 
+      begin
+        Stripe::Charge.create(card: params[:stripeToken], amount: 999, description: "Myflix charge for #{@user.email}", currency: 'usd') unless Rails.env.test?
```

Comment:
actually I could do this too. The charge should be created only for if there is Stripe configuration.

```
Stripe::Charge.create(card: params[:stripeToken], amount: 999, description: "Myflix charge for #{@user.email}", currency: 'usd') 
if Stripe.api_key
```